src/partials/footer.html

```diff
@@ -0,0 +1,9 @@
+<footer class="footer">
+  <p class="footer__item">
+    © 2023 | All Rights Reserved | Developed with
+    <svg class="heart-icon" width="14" height="12.75">
+      <use href="./images/icons.svg#icon-heart"></use>
+    </svg>
+    by<button type="button" class="footer__button" data-modal-open="">GoIT Students</button>
+  </p>
```

Comment:
I suggest make it inside a container because if we gonna add something inside our footer it would be helpful. It's not necessary right now but keep it in mind.